src/content/docs/faq.md

```diff
@@ -451,3 +451,48 @@ If you want to know more, we recommend reading
 [this excellent article by @orhun](https://blog.orhun.dev/stdout-vs-stderr/).
 
 [`BufWriter`]: https://doc.rust-lang.org/std/io/struct.BufWriter.html
+
+## How do I avoid panics due to out of range calls on the Buffer?
+
+In general, most of the code in tui-rs and hence Ratatui was not designed around using `Result`s to
+prevent panics. We have an
```

Comment:
When dealing with writing to the buffer and out of bounds panics specifically, right?

Reply:
There are 1.6k functions in Ratatui. 36 of them return Result.

Reply:
Got it.